### DISCUSSION TITLE
How to get value of 2 DOM element and compare the value in automa

### DISCUSSION THREAD

**Discussion by r0nny** [AutomaApp/automa#91] [Q&A]
How can I compare 2 DOM element using built-in block? If let say we need to do it using custom javascript block, is there such documentation for custom javascript?

**Comment by Kholid060** [comment]
You only need to insert your javascript code into the javascript block and call `automaNextBlock` inside the code to execute the next block. You can see the detail in the [blocks docs](https://github.com/Kholid060/automa/wiki/Blocks#javascript-code).
